python/python/lance/dataset.py

```diff
@@ -419,8 +419,9 @@ def create_index(
         self._ds.create_index(column, index_type, name, metric, kwargs)
         return LanceDataset(self.uri)
 
-    def _create_version_from_fragments(
-        self,
+    @staticmethod
+    def _commit(
```

Comment:
ok so basically the intended usage here is:

1. each executor node create a new fragment
2. each fragment gets written to gs bucket under the lance directory's /data subdir
3. call this _commit to create a) a new manifest file and b) update the _latest.manifest file?

Reply:
and these fragments can either be created from scratch or appending a column to an existing fragment?